plugins/utils/auto-session.nix

```diff
@@ -0,0 +1,187 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.auto-session;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.auto-session =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "auto-session";
+
+      package = helpers.mkPackageOption "auto-session" pkgs.vimPlugins.auto-session;
+
+      logLevel =
+        helpers.defaultNullOpts.mkEnum
+        ["debug" "info" "warn" "error"]
+        "error"
```

Comment:
Upstream default seems to be info ?

Reply:
This line suggests `"error"` no ?
https://github.com/rmagatti/auto-session/blob/7afbb149f87be279778689596c781882014f7eef/lua/auto-session/init.lua#L52

Reply:
It seems like, the readme was telling `info` though....